snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/SnomedRf2ExportRequest.java

```diff
@@ -251,6 +252,8 @@ void setLocales(List<ExtendedLocale> locales) {
 
 	@Override
 	public ExportResult execute(final RepositoryContext context) {
+		// register export start time for later use
+		final long exportStartTime = Instant.now().toEpochMilli();
```

Comment:
How about using the export reference branch's head timestamp? We could even run the export on a `BranchContext` just to ensure that it's not a moving target.

Reply:
Great idea, but I'm afraid we cannot change the `RepositoryContext` to `BranchContext` in the `SnomedRF2ExportRequest` now. Feel free to experiment with it.